### DISCUSSION TITLE
How to change the default window size of any application?

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1681] [Q&A] [answered]
I want to change the default window size of all the application

**Comment by kRHYME7** [comment]
Please seek the Hyorland wiki

**Comment by sanjyay** [comment]
We have to specific command for each application. I wanted to know if there was a way to generalize it for whatever application that is opened ?

**Comment by kRHYME7** [comment]
Using regex like this .*  
this will match all

**Comment by sanjyay** [comment, marked as answer]
In .config/hypr/themes,edit theme.conf to alter window size of any application
